.ci/release-github.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
what do you think to add a brief comment about what's the goal of this script?

Reply:
https://github.com/elastic/apm-aws-lambda/pull/362/commits/26ad58d73304cd8bdd2270f2f250f51b1076c10c